systemd/systemd.go

```diff
@@ -43,18 +54,20 @@ func FindClamdProcess() ctypes.Prod {
 	var out bytes.Buffer
 	cmd.Stdout = &out
 
+	noClamAV := ctypes.Prod{}
+
 	if err := cmd.Run(); err != nil {
-		return ctypes.Prod{}
+		return noClamAV
 	}
 
 	lines := strings.Split(out.String(), "\n")
 
-	for _, line := range lines {
-		if strings.Contains(line, "clamd") {
-			cd, err := GetClamDetails()
-			if err != nil {
-				return ctypes.Prod{}
-			}
+	isRunning := slices.ContainsFunc(lines, func(process string) bool {
+		return strings.Contains(process, "clamd")
+	})
```

Comment:
arguably, we don't really need to loop thru lines, we could just look for the small string in the big string :-) but at least ContainsFunc exits early now.